src/pubsub/pubsub_index.erl

```diff
@@ -41,7 +41,18 @@ init() ->
                          {attributes, record_info(fields, pubsub_index)}]).
 
 new(Index) ->
-    case mnesia:read({pubsub_index, Index}) of
+    Parent = self(),
```

Comment:
This is not needed if we don't use `worker_pool` for mnesia

Reply:
fixed